business_requirement/views/business_view.xml

```diff
@@ -159,7 +159,7 @@
                                 <div class="oe_kanban_bottom_right">
                                     <div class="pull-left" groups="base.group_user">
                                         <img
-                                            t-att-src="kanban_image('res.users', 'image_small', record.responsible_user_id.raw_value)"
+                                            t-att-src="kanban_image('res.users', 'image_128', record.responsible_user_id.raw_value)"
```

Comment:
I'm not doubting. Shouldn't this be `image_64`?

Reply:
The smallest size is `image_128` (`image_64` does not exist).

Reply:
The `width` and `height` parameters were previously defined.

https://github.com/OCA/business-requirement/blob/8263e15bfaf5c360ff75e08080e62242ce318f4b/business_requirement/views/business_view.xml#L161-L168

Reply:
Ah, OK